components/reddit/reddit.app.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  type: "app",
+  app: "reddit",
+  methods: {
+    _accessToken() {
+      return this.$auth.oauth_access_token;
+    },
+    _apiUrl() {
+      return `https://oauth.reddit.com`;
+    },
+    async _makeRequest(opts) {
+      if (!opts.headers) opts.headers = {};
+      opts.headers.authorization = `Bearer ${this._accessToken()}`;
+      opts.headers["user-agent"] = "@PipedreamHQ/pipedream v0.1";
+      const { path } = opts;
+      delete opts.path;
+      opts.url = `${this._apiUrl()}${path[0] === "/" ? "" : "/"}${path}`;
+      return await require("@pipedreamhq/platform").axios(this, opts);
```

Comment:
can you use plain old `axios` as we discussed in Slack?

Reply:
sure.